config/rbac/service_account.yaml

```diff
@@ -1,3 +1,4 @@
+---
```

Comment:
Please remove these changes from this PR.

Reply:
done.